drivers/bluetooth/CMakeLists.txt

```diff
@@ -4,3 +4,4 @@ zephyr_library()
 zephyr_library_property(ALLOW_EMPTY TRUE)
 zephyr_library_include_directories(${ZEPHYR_BASE}/subsys/bluetooth)
 add_subdirectory(hci)
+add_subdirectory(custom)
```

Comment:
custom is a strange name?

Reply:
The reason for the name custom is that this driver is not HCI based, unlike the existing bluetooth drivers within zephyr. There is existing entries in the configs for custom drivers, so we simply added a folder for custom drivers and located our driver within that folder.